lua/cfc_prop_restoration/core/sv/sv_init.lua

```diff
@@ -5,6 +5,10 @@ local expireTime = 600     -- Time (in seconds) for the player to reconnect befo
 local autosaveDelay = 180  -- How often (in seconds) the server saves prop data
 local nextSave = CurTime() + autosaveDelay
 
+local areacopy_classblacklist = {
+    gmod_anchor = true
+}
```

Comment:
It might help to include a comment about what this blacklist is for, and what kinds of entities might be put into it